helpers/browser/driver.js

```diff
@@ -90,18 +93,35 @@ class ChromeProtocol {
     this._chrome.on('event', req => _log('verbose', '<=', req));
   }
 
-  // bind listeners for protocol events
+  /**
+   * Bind listeners for protocol events
+   * @param {!string} eventName
+   * @param {!Function} cb
+   */
   on(eventName, cb) {
     if (this._chrome === null) {
       throw new Error('Trying to call on() but no cri instance available yet');
     }
     // log event listeners being bound
     _log('info', 'listen for event =>', {method: eventName});
-
     this._chrome.on(eventName, cb);
   }
 
-  // call protocol methods
+  /**
+   * Unbind event listeners
+   * @param {!string} eventName
+   * @param {!Function} cb
```

Comment:
It doesn't appear to be in the external styleguide, but it's in the internal one. `Function` is typed as just an instance of the type `Function`, with no other information, while `function(whatever): whatever` provides more information on its parameters and return type. In this case it's not much information :) just that it's variadic and the parameters could be anything, but that's on EventEmitter being so wishy-washy, not us